kubernetes/assets/dashboards/kubernetes_dashboard.json

```diff
@@ -962,7 +962,7 @@
                 "type": "timeseries",
                 "requests": [
                     {
-                        "q": "sum:kubernetes.io.read_bytes{$scope,$daemonset,$service,$namespace,$label,$cluster,$deployment,$node} by {host}",
+                        "q": "sum:kubernetes.io.read_bytes{$scope,$deployment,$statefulset,$replicaset,$daemonset,$label,$cluster,$node} by {host}",
```

Comment:
is `namespace` not needed here and below)?

Reply:
good catch